src/Symfony/Component/Config/Definition/PrototypedArrayNode.php

```diff
@@ -119,7 +119,7 @@ public function setAddChildrenIfNoneSet(int|string|array|null $children = ['defa
     }
 
     /**
-     * The default value could be either explicited or derived from the prototype
+     * The default value could be either explicit or derived from the prototype
```

Comment:
fabbot's codespell has detected this typo, but could not update it automatically because there is 2 replacements in its [dictionary](https://github.com/codespell-project/codespell/blob/87a280039336c73d45faba7829aea860e68de4a3/codespell_lib/data/dictionary.txt#L27399).